Loop/Managers/TestingScenariosManager.swift

```diff
@@ -327,7 +327,9 @@ extension TestingScenariosManager {
         case .success(let setupUIResult):
             switch setupUIResult {
             case .createdAndOnboarded(let cgmManager):
-                return cgmManager as! TestingCGMManager
+                let cgmManager = cgmManager as! TestingCGMManager
+                cgmManager.autoStartTrace = false
```

Comment:
I would have figured this would default to `true` and override to `false` as needed. I won't block the PR, but curious why it defaults to `false`?

Reply:
This Should be defaulting to true. This is the override here where we reload the CGMManager before the scenario trace is loaded. Overriding to false here stops the initial trace from running.